src/entities/trades/curve/CurveTrade.ts

```diff
@@ -325,7 +322,7 @@ export class CurveTrade extends Trade {
           amountInBN.toString(),
           exchangeRoutingInfo.routes,
           exchangeRoutingInfo.indices,
-          exchangeRoutingInfo.expectedAmountOut.mul(98).div(100).toString(),
+          exchangeRoutingInfo.expectedAmountOut.mul(0.98).toString(),
```

Comment:
Does BigNumber library support float numbers?

Reply:
that as well!